lib/galaxy/webapps/galaxy/api/tool_dependencies.py

```diff
@@ -193,12 +193,20 @@ def summarize_toolbox(self, trans: ProvidesAppContext, **kwds):
         """
         GET /api/dependency_resolvers/toolbox
 
-        Summarize requirements across toolbox (for Tool Management grid). This is an experiemental
-        API particularly tied to the GUI - expect breaking changes until this notice is removed.
+        Summarize requirements and their dependencies across toolbox (for Tool Management grid).
+        This is an experimental API particularly tied to the GUI - expect
+        breaking changes until this notice is removed.
 
         Container resolution via this API is especially experimental and the container resolution
```

Comment:
Isn't that part sufficient ?

Reply:
In my opinion the current help is completely insufficient (and I consider my additions only a small step forward). I don't know any other API docs, but in my opinion the goal should be that one reads the documentation and then knows what the function is doing (without studying any code or similar). 

Clearly this is difficult (maybe impossible) .. but we may try.

Reply:
I don't think API docstrings should give you implementation details.